### PR TITLE
[fastlane] add locale envs

### PR DESCRIPTION
**Pull request by hjanuschka**
as proposed here: https://github.com/fastlane/fastlane/issues/6914

i actually added my own LANG as ✅  because getting ⛔️ just because not an american is too hard for me.

### CODE REVIEW

File: fastlane/lib/fastlane/environment_printer.rb [hunk 1]
```diff
@@ -15,6 +15,7 @@ def self.get
       require "fastlane/markdown_table_formatter"
       env_output = ""
       env_output << print_system_environment
+      env_output << print_system_locale
```
Comment (KrauseFx) [inline fastlane/lib/fastlane/environment_printer.rb]: Isn't that something we could fix, instead of just showing in the output?

**Comment by hjanuschka** [comment]
yeah i am not sure what happens once you enforce en_US.utf8 - on a system wich hasn't installed the locale.

for debugging/troubleshooting, having it in `fastlane env` is nice anyway.
(not 100% sure about the emojis, there are a lot of charsets without problems - that are not US)

**Comment by milch** [comment]
I agree, I'm not sure about all the things that could possibly break if we just set the locale for the user. At least now we know the error message, it's easy to debug, and if we spot that the locale is set incorrectly in the `env` output (after merging this PR) it'll be easy to close issues that have this problem as well.

File: fastlane/lib/fastlane/environment_printer.rb [hunk 2]
```diff
@@ -128,6 +129,25 @@ def self.print_loaded_gems
       return env_output
     end
 
+    def self.print_system_locale
+      env_output = "### System Locale\n\n"
+      table = "| Variable | Value |  |\n"
+      table << "|-----|---------|----|\n"
+      ["LANG", "LC_ALL", "LANGUAGE"].each do |e|
+        env_icon = "⛔️"
+        if ["en_US.UTF-8", "de_DE.UTF-8"].include?(ENV[e]) || !ENV[e]
```
Comment (milch) [inline fastlane/lib/fastlane/environment_printer.rb]: How about we just test that it is ending in `.UTF-8` instead? I've read reports from users using it with all kinds of UTF-8 locales and it fixed their issues.

**Review by milch — CHANGES_REQUESTED** [review]

**Comment by milch** [comment]
I think we should merge this for now since I think it's going to be useful, we can always revert it later if we decide to just set the environment ourselves. There's some other code we'd need to remove too, anyway.

**Comment by hjanuschka** [comment]
yeah i think it is not bad to have locale info in issues - once we decide to set it on startup our self,
i am happy to do a PR and remove the code or rework it - to print all `FASTLANE_|FL_|DELIVER....` env (only if they exists, without value for sure 👍 )


@milch ok i redid it with `end_with?` - UTF8

and removed the broken sign on undefined envs.
and also check if atleast one of the vars has UTF8 otherwise output this table:

### System Locale

| Error                       |
| --------------------------- |
| No Locale with UTF8 found 🚫 |

**Review by milch — APPROVED** [review]

**Comment by milch** [comment]
Thanks, looks good!

**Pull request merged**